appstudio-utils/util-scripts/lib/fetch/rekor.sh

```diff
@@ -57,8 +57,8 @@ rekor-log-entry-save() {
   local rekor_host=${2:-rekor.sigstore.dev}
 
   local entry_data=$( rekor-log-entry $log_index $rekor_host )
-  local entry_file=$( json-data-file rekor $rekor_host index $log_index entry )
-  local att_file=$( json-data-file rekor $rekor_host index $log_index attestation )
+  local entry_file=$( json-input-file rekor $rekor_host index $log_index entry )
+  local att_file=$( json-input-file rekor $rekor_host index $log_index attestation )
```

Comment:
We won't fetching data from rekor in the short/medium term, so better not to change this code.